packages/common/src/types.ts

```diff
@@ -1,5 +1,5 @@
 import { z } from 'zod';
-import { languageCodeToLanguage } from './languages';
+import { LanguageCode, languageCodeToLanguage } from './languages';
```

Comment:
I think your zod enum means we can get rid of `zodLanguageCode` just below which does the same thing (and also `getKeys`). Your solution of making a zod enum from the array of country codes is way simpler than what I did!